modules/Story/Story.tsx

```diff
@@ -6,7 +6,6 @@ import Image from '@prezly/uploadcare-image';
 import classNames from 'classnames';
 import dynamic from 'next/dynamic';
 
-import { StoryStickyBar } from '@/components';
```

Comment:
If this component is no longer used, let's remove it.

Reply:
Removed